recipe/meta.yaml

```
@@ -14,6 +14,9 @@ source:
   patches:
     - remove-xxd-dep.patch
 
+{% if mpi == 'nompi' %}
```

Might require double quotes instead of single quotes. Change this in case build hash does not contain 102 as build number.
```suggestion
{% if mpi == "nompi" %}
```